titus-api/src/main/java/com/netflix/titus/api/jobmanager/model/job/vpc/IpAddress.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright 2019 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.titus.api.jobmanager.model.job.vpc;
+
+import java.util.Objects;
+import javax.validation.constraints.Size;
+
+/**
+ * IP Address to be assigned to task.
+ */
+public class IpAddress {
```

Comment:
I had the internal classes follow the same structure as the proto messages. I'm OK having them differ but since I'm not the one who introduced the proto messages in the first place, I'd make changes here to the internal types and then leave the proto types for another discussion.